### PR TITLE
Style improvements

### PR DESCRIPTION
**Pull request by palashkaria**
some minor tweaks to the styles;
- Adds hover state, closer to Roam text underline; and card background on hover
- Better checkbox focus/hover states
- some alignment fixes

Attached:  GIF with horrible quality for comparison :) 

Screen Recording 2020-05-12 at 10 22 17 PM

### CODE REVIEW

**Comment by palashkaria** [comment]
@Stvad please take a look when you can :)

**Comment by Stvad** [comment]
Nice!

File: src/ts/components/Checkbox.tsx [hunk 1]
```diff
@@ -9,6 +10,11 @@ type CheckboxProps = {
     onSave: (newValue: boolean) => void
 }
 
+const Label = styled('label')`
```
Comment (Stvad) [inline src/ts/components/Checkbox.tsx:13]: can we name it something else to make it clear that it's our custom component?

File: src/ts/containers/Settings/FeatureConfig.tsx [hunk 2]
```diff
@@ -90,14 +90,15 @@ export const FeatureConfig = ({feature}: FeatureConfigProps) => {
     )
 }
 
-const FeatureConfigContainer = styled('div')`
-    padding-left: 30px;
-`
+const FeatureConfigContainer = styled('div')``
```
Comment (Stvad) [inline src/ts/containers/Settings/FeatureConfig.tsx:93]: should we just use div, or is it considered a good practice to use named components?

Comment (palashkaria) [reply src/ts/containers/Settings/FeatureConfig.tsx:93]: just easier to find the tree; Plus easier to add styles if needed

**Pull request merged**